ejb3/src/main/java/org/jboss/as/ejb3/remote/AssociationImpl.java

```diff
@@ -217,6 +230,34 @@ public CancelHandle receiveInvocationRequest(@NotNull final InvocationRequest in
         return cancellationFlag::cancel;
     }
 
+    private static <A> boolean intersect(final Set<A> fst, final Set<A> snd) {
+        fst.retainAll(snd);
+        return !fst.isEmpty();
+    }
+
+    private static Set<String> typesOf(final Object o) {
```

Comment:
CI showed problems with Weld previously which were fixed by creating the exclusion list, then it went all green. 
I wonder what TCK tests do which is not included in about 6k tests we run for 6. I am also writing a mail to our RC to ask for TCK6 pass to see if the same issue appears in older version.